### ISSUE TITLE
Getting Error -> W18: Invalid character in group name 

### ISSUE THREAD

**Issue by theCollectiv** [tanvirtin/monokai.nvim#34]
I'm using monokai.nvim for a long time and enjoying it. I haven't changed my setup (as far as I know), but now I get this error-message 
```
Error detected while processing nvim_exec() called at /home/metamorph/.local/share/nvim/site/pack/packer/start/monokai.nvim/colors/monokai.vim:1:                           
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
W18: Invalid character in group name                                                                                                                                        
-- More -- 
...
```
After hitting a Enter a view times, neovim is using monokai normally. Maybe it's just me, but if someone got the problem too and got a solution, I'd be glad to know ;-)

PS.: 
```
 >> nvim --version
NVIM v0.8.0-dev
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
```

```
 >> lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 22.04.1 LTS
Release:        22.04
Codename:       jammy
```

**Comment by theCollectiv** [comment]
I got the same problem with (at least an)other colorscheme(s)* too. Maybe it's some weird change in the neovim-dev-branch.

* same error: ellisonleao/gruvbox.nvim / no error: kartikp10/noctis.nvim

**Comment by theCollectiv** [comment]
Solution: Had to update to the latest version, seems to be an (ubuntu)-ppa-problem:
Switched from:
(the stable Version [NVIM v0.8.0-dev] seems out of date [2022-03-24] and something is breaking the newest versions of plugins)
```
[Neovim Stable : “Neovim PPA” team](https://launchpad.net/~neovim-ppa/+archive/ubuntu/stable)
```
to neovim-nightly
```
[Neovim Unstable : “Neovim PPA” team](https://launchpad.net/~neovim-ppa/+archive/ubuntu/unstable)
```
and everything works fine and as expected.

**Comment by Walther** [comment]
This seems to also happen on Debian Bookworm, where the latest available neovim version is `0.7.2`.
Is there possibly a workaround for this - other than installing neovim from the source?
